.nxreleaserc.json

```
@@ -0,0 +1,7 @@
+{
+  "changelog": true,
+  "npm": true,
```

Como não iremos subir pacotes pelo npm, podemos deixar essa opção como `false`.

```suggestion
  "npm": false,
```